recipe/migrations/python310.yaml

```diff
@@ -0,0 +1,30 @@
+migrator_ts: 1634137107
+__migrator:
+    migration_number: 1
+    operation: key_add
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy  # new entry
+    paused: false
+    longterm: True
+    pr_limit: 50
+    exclude:
+      # this shouldn't attempt to modify the python feedstocks
+      - python
+      - pypy3.6
+      - pypy-meta
+
+python:
+  - 3.10.* *_cpython
+# additional entries to add for zip_keys
+numpy:
+  - 1.19

```

Comment:
The dependency resolution will block in any case if you need a numpy and we haven't built it yet.  Since this is a zip_keys set we HAVE to have a value in here, so may as well pick the same one we are using for the rest of them.

There are a bunch of packages that we'll build before we get to a numpy in any case.  

If we need to bump the numpy version in case we cannot build 1.19 for 3.10 at all we can just release a new migrator version.

Reply:
Ohhhh. Yes, we'd need to nackport that build or add an ABI branch.

Reply:
> @mariusvniekerk: If we need to bump the numpy version in case we cannot build 1.19 for 3.10 at all we can just release a new migrator version.

Numpy will only support 3.10 [earliest](https://github.com/numpy/numpy/pull/20098) from 1.21.3, so backporting to 1.19 sounds tricky.

Reply:
For the record, +1 to using `1.21`. Note that `1.21.3` is not yet out (that's the first version that will have wheels for 3.10) but `1.21.2` will build just fine with Python 3.10.

There's no point backporting fixes to `1.20.x` or earlier, that's just extra effort and means things get out of sync with the officially supported releases and with PyPI.